spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -31,50 +31,50 @@ content into your application; rather pick only the properties that you need.
 	# BANNER
 	banner.charset=UTF-8 # Banner file encoding.
 	banner.location=classpath:banner.txt # Banner file location.
-	banner.image.location=classpath:banner.gif # Banner image file location (jpg/png can also be used).
+	banner.image.location=classpath:banner.gif # Banner image file location (jpg or png can also be used).
```

Comment:
@Buzzardo this description is actually used in the IDE (the metadata appendix you've also reviewed, thanks!). So if we change anything in here we have to change the source code. This content is actually a copy of the Javadoc of the related property. 

Given the number of changes here it will take some time to process.

Reply:
This is another unfortunate casualty of having the same information in two places. Can you bite the bullet on auto-generating the appendix?